rules/S7137/secrets/metadata.json

```
@@ -0,0 +1,56 @@
+{
+  "title": "RubyGems.org API keys should not be disclosed",
```

```suggestion
  "title": "RubyGems API keys should not be disclosed",
```

We never used the URL for any secret name, so we shouldn't start now.